app/components/settings_drawer/settings_drawer.js

```diff
@@ -21,6 +21,7 @@ import SafeAreaView from 'app/components/safe_area_view';
 import Drawer from 'app/components/drawer';
 import UserStatus from 'app/components/user_status';
 import {NavigationTypes} from 'app/constants';
+import {confirmOutOfOfficeDisabled} from 'app/utils/status';
```

Comment:
inline require?

Reply:
@enahum as for the OOF work being included as an inline-require.  I'm not sure it's necessary.

It's not crucial to the cold start experience